remo/domain/image.py

```diff
@@ -147,3 +166,12 @@ def view_annotate(self, annotation_set_id: int):
             annotation_set_id: annotation set id
         """
         return self.sdk.view_annotate_image(annotation_set_id, self.id)
+
+
+class AnnotatedImage(Image):
```

Comment:
I'm not sure about introducing AnnotatedImage, but if not we would have to think about a way to combine images with annotations when there are multiple annotation sets

Reply:
ok, I can think about integrating Annotations into Image

Reply:
@vmanilo I think we can leave this for later, and merge for now. It requires understanding better the use cases, it's not something immediate I reckon

Reply:
ok, sure